### PR TITLE
Renaming and moving text

### PR DESCRIPTION
**Pull request by thothmoses**
Moved some text around - renamed some sections - injected optional notes within the tutorial.

References #xxxx

### CODE REVIEW

File: docs-source/docs/modules/microservices-tutorial/pages/complete-entity.adoc [hunk 1]
```diff
@@ -412,7 +412,7 @@ When finished, stop the service with `ctrl-c`. Leave PostgresSQL running for the
 ----
 docker-compose stop
 ----
-
+NOTE: The following steps for cloud deloyment are optional. If you are only deploying locally you can skip to the next section of the tutorial.
```
Comment (ignasi35) [inline docs-source/docs/modules/microservices-tutorial/pages/complete-entity.adoc:415]:
```suggestion
NOTE: The following steps for cloud deloyment are optional. If you are only running locally you can skip to the next section of the tutorial.
```

`deploying locally` can be misleading if users are trying our `microk8s` or `minikube`.

File: docs-source/docs/modules/deployment/pages/index.adoc [hunk 2]
```diff
@@ -7,3 +7,32 @@ The Akka Cloud Platform enables developers to quickly build and deploy cloud-nat
 Akka Platform technology enables you to create microservices that are resilient to failure, highly efficient, and operate at any scale. Simple integration paths provide streamlined integration with relevant services in the ecosystem.
 
 All necessary building blocks such as persistence, event sourcing, distributed data, streaming, telemetry and observability are included. It also provides simple integration paths for relevant cloud services (e.g. Kafka, Cassandra, data stores).
+
+== Getting Started with Cloud Deployment
+Before you can get started with cloud deployment you should:
+
+. Determine which platform you will be using (Google Cloud Platform, for example).
+. Have an active account with your chosen platform.
+. Subscribe to the Akka Cloud Platform with xref:gcp-install.adoc[Google] or xref:aws-install.adoc[Amazon].
```
Comment (ignasi35) [inline docs-source/docs/modules/deployment/pages/index.adoc:16]:
```suggestion
. Subscribe to the Akka Cloud Platform with xref:gcp-install.adoc[GCP] or xref:aws-install.adoc[AWS].
```

I have no idea what the correct name is on either case. Maybe one for @jtownley to help shed some light?

File: docs-source/docs/modules/deployment/pages/index.adoc [hunk 3]
```diff
@@ -7,3 +7,32 @@ The Akka Cloud Platform enables developers to quickly build and deploy cloud-nat
...
+
+- TLS Certificates — TLS certificates are required for external access to applications managed by Akka Cloud Platform. You will find the pertinent information within the xref:deployment:aws-ingress.adoc#_tls_certificate[AWS LoadBalancer Controller] page.
+
+- Support — Support for the Akka Cloud Platform requires registration. https://aws.amazon.com/marketplace/pp/B08TLV16XM#pdp-support[Details] regarding support, and more, can be found on the AWS Marketplace.
+
+- License expiration and renewal — You may have purchased your license by subscribing to the Akka Cloud Platform product on the AWS Marketplace, or  directly from Lightbend. In either case, the license expiration and renewal for the Akka Cloud Platform is handled as follows:
+* If the Akka Cloud Platform license expires without being renewed, your current application will continue to run, but you will no longer be able to deploy or update.
+* If you have acquired a term license directly from Lightbend, and it expires, your license will automatically revert into a pay-as-you-go license.
+* When you need to renew your license, please contact mailto:aws@lightbend.com[] with your request.
```
Comment (ignasi35) [inline docs-source/docs/modules/deployment/pages/index.adoc:38]: Why is there an AWS-specific section on this `index.adoc`? Should there be a similar section for GCP? Or maybe this AWS-specific section must be moved to an AWS index page?

**Review by ignasi35 — APPROVED** [review]
Left a few minor comments.

File: docs-source/docs/modules/deployment/pages/index.adoc (hunk 3 above)
Comment (thothmoses) [reply docs-source/docs/modules/deployment/pages/index.adoc:38]: I will be dding GCP specific as well.

File: docs-source/docs/modules/microservices-tutorial/pages/projection-grpc-client.adoc [hunk 4]
```diff
@@ -297,7 +297,7 @@ When finished:
 ----
 docker-compose stop
 ----
-NOTE: The following steps for cloud deloyment are optional. If you are only deploying locally you can skip to the next section of the tutorial.
+NOTE: The following steps for cloud deloyment are optional. If you are only running locally you have completed the tutorial Congratulations.
```
Comment (ignasi35) [inline docs-source/docs/modules/microservices-tutorial/pages/projection-grpc-client.adoc:300]:
```suggestion
NOTE: The following steps for cloud deloyment are optional. If you are only running locally you have completed the tutorial. Congratulations.
```

File: docs-source/docs/modules/microservices-tutorial/pages/complete-entity.adoc [hunk 5]
```diff
@@ -412,7 +412,7 @@ When finished, stop the service with `ctrl-c`. Leave PostgresSQL running for the
 ----
 docker-compose stop
 ----
-NOTE: The following steps for cloud deloyment are optional. If you are only deploying locally you can skip to the next section of the tutorial.
+NOTE: The following steps for cloud deloyment are optional. If you are only running locally you can skip to the next section of the tutorial.
```
Comment (ignasi35) [inline docs-source/docs/modules/microservices-tutorial/pages/complete-entity.adoc:415]: Just realized that `deloyment` is misspelled in all the occurrences.

File: docs-source/docs/modules/microservices-tutorial/pages/projection-grpc-client.adoc (hunk 4 above)
Comment (ignasi35) [reply docs-source/docs/modules/microservices-tutorial/pages/projection-grpc-client.adoc:300]: A missing `.` and the `deloyment`.

File: docs-source/docs/modules/microservices-tutorial/pages/complete-entity.adoc (hunk 5 above)
Comment (thothmoses) [reply docs-source/docs/modules/microservices-tutorial/pages/complete-entity.adoc:415]: ack - fixing

File: docs-source/docs/modules/microservices-tutorial/pages/projection-grpc-client.adoc (hunk 4 above)
Comment (thothmoses) [reply docs-source/docs/modules/microservices-tutorial/pages/projection-grpc-client.adoc:300]: fixing

**Pull request merged**